src/py42/sdk/queries/query_filter.py

```diff
@@ -190,14 +215,37 @@ class FilterGroup(object):
 
     When :func:`str()` is called on a :class:`FilterGroup` instance, the combined filter items are
     transformed into a JSON string to be used as part of a Forensic Search or Alert query.
+
+    When :func:`dict()` is called on a :class:`FilterGroup` instance, the combined filter items are
+    transformed into the Python `dict` equivalent of their JSON representation. This can be useful
+    for programmatically manipulating a :class:`FilterGroup` after it's been created.
     """
 
     def __init__(self, filter_list, filter_clause=u"AND"):
         self._filter_list = filter_list
         self._filter_clause = filter_clause
 
+    @classmethod
+    def from_dict(cls, _dict, filter_clause=u"AND"):
+        filter_list = [QueryFilter.from_dict(item) for item in _dict[u"filters"]]
+        return cls(filter_list, filter_clause=filter_clause)
+
+    @property
```

Comment:
I exposed these so that they can be explored more easily when iterating through a list of filter groups and trying to do logic based on them